<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout"/>
    </item>
    <item row="1" column="0">
     <widget class="QScrollArea" name="scrollArea">
      <property name="minimumSize">
       <size>
        <width>800</width>
        <height>0</height>
       </size>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="ResultDrawingBoard" name="mainImageResult">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>798</width>
         <height>716</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QFrame" name="frame">
      <property name="minimumSize">
       <size>
        <width>180</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="resolvingButton">
           <property name="text">
            <string>解算</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="amplifyButton">
           <property name="text">
            <string>放大</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="reduceButton">
           <property name="text">
            <string>缩小</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="attitudeButton">
           <property name="text">
            <string>属性</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QPushButton" name="partitionButton">
           <property name="text">
            <string>局部</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="fitButton_2">
           <property name="text">
            <string>适应</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="toolTab">
          <attribute name="title">
           <string>Tab 1</string>
          </attribute>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QPushButton" name="drawLineButton">
               <property name="text">
                <string>直线</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="drawAngleButton">
               <property name="text">
                <string>夹角</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="drawRectangleButton">
               <property name="text">
                <string>矩形</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="drawEllipticButton">
               <property name="text">
                <string>椭圆</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="drawOccurrenceButton">
               <property name="text">
                <string>产状</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="drawPolygonButton">
               <property name="text">
                <string>多边形</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="reportTab">
          <attribute name="title">
           <string>Tab 2</string>
          </attribute>
          <layout class="QHBoxLayout" name="horizontalLayout_5"/>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>文件（F）</string>
    </property>
    <addaction name="importAction"/>
   </widget>
   <widget class="QMenu" name="menuTool">
    <property name="title">
     <string>工具（T）</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>视图（V）</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>帮助（H）</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTool"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionTest">
   <property name="text">
    <string>Test</string>
   </property>
  </action>
  <action name="actionTest_2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Test</string>
   </property>
  </action>
  <action name="actionTest_3">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Test</string>
   </property>
  </action>
  <action name="actionTest3">
   <property name="text">
    <string>Test3</string>
   </property>
  </action>
  <action name="actionTest3_2">
   <property name="text">
    <string>Test3</string>
   </property>
  </action>
  <action name="importAction">
   <property name="text">
    <string>导入（I）</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ResultDrawingBoard</class>
   <extends>QWidget</extends>
   <header>result_drawing_board.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
